scripts/scripts.js

```diff
@@ -9,7 +9,7 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
-/* global sessionStorage Image */
+/* global sessionStorage Image fetch PerformanceObserver performance */
```

Comment:
I think we should switch the `eslint` config to use the `browser` environment. That would make the tedious task of declaring global functions in every file obsolete.